<ui version="4.0" >
 <class>MuonAnalysis</class>
 <widget class="QWidget" name="MuonAnalysis" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_8" >
   <property name="margin" >
    <number>1</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="Home" >
      <attribute name="title" >
       <string>Home</string>
      </attribute>
      <widget class="QWidget" name="horizontalLayoutWidget_4" >
       <property name="geometry" >
        <rect>
         <x>430</x>
         <y>600</y>
         <width>123</width>
         <height>26</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4" >
        <item>
         <spacer name="horizontalSpacer" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="exitButton" >
          <property name="text" >
           <string>Exit</string>
          </property>
          <property name="checkable" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>10</y>
         <width>551</width>
         <height>121</height>
        </rect>
       </property>
       <property name="title" >
        <string>Instrument</string>
       </property>
       <widget class="QLabel" name="label_3" >
        <property name="geometry" >
         <rect>
          <x>110</x>
          <y>20</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text" >
         <string>Time Zero (ns):</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="timeZeroFront" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="geometry" >
         <rect>
          <x>230</x>
          <y>20</y>
          <width>31</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text" >
         <string>200</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4" >
        <property name="geometry" >
         <rect>
          <x>110</x>
          <y>50</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text" >
         <string>First Good Data (ns):</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="firstGoodBinFront" >
        <property name="geometry" >
         <rect>
          <x>230</x>
          <y>50</y>
          <width>31</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text" >
         <string>300</string>
        </property>
       </widget>
       <widget class="QLabel" name="instrumentDescription" >
        <property name="geometry" >
         <rect>
          <x>110</x>
          <y>80</y>
          <width>421</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text" >
         <string>Description:</string>
        </property>
       </widget>
       <widget class="MantidQt::MantidWidgets::InstrumentSelector" name="instrSelector" >
        <property name="geometry" >
         <rect>
          <x>10</x>
          <y>20</y>
          <width>74</width>
          <height>20</height>
         </rect>
        </property>
        <property name="techniques" stdset="0" >
         <stringlist>
          <string>Muon spectroscopy</string>
         </stringlist>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_2" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>140</y>
         <width>551</width>
         <height>111</height>
        </rect>
       </property>
       <property name="title" >
        <string>Data Files</string>
       </property>
       <widget class="QPushButton" name="pushButton" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="geometry" >
         <rect>
          <x>10</x>
          <y>20</y>
          <width>101</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text" >
         <string>Load current run</string>
        </property>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget_2" >
        <property name="geometry" >
         <rect>
          <x>10</x>
          <y>50</y>
          <width>501</width>
          <height>51</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="fileInputLayout" >
         <item>
          <widget class="MantidQt::MantidWidgets::MWRunFiles" native="1" name="mwRunFiles" >
           <property name="findRunFiles" stdset="0" >
            <bool>true</bool>
           </property>
           <property name="label" stdset="0" >
            <string>Load run</string>
           </property>
           <property name="multipleFiles" stdset="0" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_3" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>260</y>
         <width>551</width>
         <height>121</height>
        </rect>
       </property>
       <property name="title" >
        <string>Detector Grouping</string>
       </property>
       <widget class="QLabel" name="label" >
        <property name="geometry" >
         <rect>
          <x>20</x>
          <y>30</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text" >
         <string>Group / Group Pair:</string>
        </property>
       </widget>
       <widget class="QLabel" name="homePeriodsLabel" >
        <property name="geometry" >
         <rect>
          <x>20</x>
          <y>80</y>
          <width>271</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text" >
         <string>Data collected in &lt;n> Periods. Plot/analyse Period:</string>
        </property>
       </widget>
       <widget class="QComboBox" name="frontGroupGroupPairComboBox" >
        <property name="geometry" >
         <rect>
          <x>130</x>
          <y>30</y>
          <width>181</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="frontAlphaLabel" >
        <property name="geometry" >
         <rect>
          <x>330</x>
          <y>30</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text" >
         <string>with alpha:</string>
        </property>
       </widget>
       <widget class="QComboBox" name="homePeriodBox1" >
        <property name="geometry" >
         <rect>
          <x>280</x>
          <y>80</y>
          <width>51</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="homePeriodBoxMath" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="geometry" >
         <rect>
          <x>350</x>
          <y>80</y>
          <width>41</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text" >
          <string>-</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>+</string>
         </property>
        </item>
       </widget>
       <widget class="QComboBox" name="homePeriodBox2" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="geometry" >
         <rect>
          <x>410</x>
          <y>80</y>
          <width>51</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="frontAlphaNumber" >
        <property name="geometry" >
         <rect>
          <x>390</x>
          <y>30</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_4" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>390</y>
         <width>551</width>
         <height>81</height>
        </rect>
       </property>
       <property name="title" >
        <string>Plot Data</string>
       </property>
       <widget class="QPushButton" name="frontPlotButton" >
        <property name="geometry" >
         <rect>
          <x>240</x>
          <y>30</y>
          <width>75</width>
          <height>24</height>
         </rect>
        </property>
        <property name="font" >
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text" >
         <string>Plot</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2" >
        <property name="geometry" >
         <rect>
          <x>20</x>
          <y>30</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text" >
         <string>Plot type:</string>
        </property>
       </widget>
       <widget class="QComboBox" name="frontPlotFuncs" >
        <property name="geometry" >
         <rect>
          <x>100</x>
          <y>30</y>
          <width>111</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_5" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>480</y>
         <width>551</width>
         <height>101</height>
        </rect>
       </property>
       <property name="title" >
        <string>Run Information</string>
       </property>
       <widget class="QTextBrowser" name="infoBrowser" >
        <property name="geometry" >
         <rect>
          <x>10</x>
          <y>20</y>
          <width>531</width>
          <height>71</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="muonAnalysisHelp" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>600</y>
         <width>20</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text" >
        <string>?</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="GroupingOptions" >
      <attribute name="title" >
       <string>Grouping Options</string>
      </attribute>
      <widget class="QPushButton" name="loadGroupButton" >
       <property name="geometry" >
        <rect>
         <x>20</x>
         <y>20</y>
         <width>111</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text" >
        <string>Load Grouping File</string>
       </property>
      </widget>
      <widget class="QPushButton" name="saveGroupButton" >
       <property name="geometry" >
        <rect>
         <x>150</x>
         <y>20</y>
         <width>111</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text" >
        <string>Save Grouping</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="pairTable" >
       <property name="geometry" >
        <rect>
         <x>20</x>
         <y>400</y>
         <width>531</width>
         <height>161</height>
        </rect>
       </property>
       <property name="rowCount" >
        <number>10</number>
       </property>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <column>
        <property name="text" >
         <string>Group Pair &#xd;
(Name)</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>Forward &#xd;
(Group name)</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>Backward &#xd;
(Group name)</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>Alpha</string>
        </property>
       </column>
      </widget>
      <widget class="QPushButton" name="clearGroupingButton" >
       <property name="geometry" >
        <rect>
         <x>280</x>
         <y>20</y>
         <width>101</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text" >
        <string>Clear Grouping</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5" >
       <property name="geometry" >
        <rect>
         <x>300</x>
         <y>330</y>
         <width>46</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text" >
        <string>Plot type:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6" >
       <property name="geometry" >
        <rect>
         <x>260</x>
         <y>570</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text" >
        <string>Plot type (for pair):</string>
       </property>
      </widget>
      <widget class="QPushButton" name="groupTablePlotButton" >
       <property name="geometry" >
        <rect>
         <x>480</x>
         <y>330</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text" >
        <string>Plot</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pairTablePlotButton" >
       <property name="geometry" >
        <rect>
         <x>480</x>
         <y>570</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text" >
        <string>Plot</string>
       </property>
      </widget>
      <widget class="QPushButton" name="guessAlphaButton" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="geometry" >
        <rect>
         <x>20</x>
         <y>570</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text" >
        <string>Guess Alpha</string>
       </property>
      </widget>
      <widget class="QComboBox" name="groupTablePlotChoice" >
       <property name="geometry" >
        <rect>
         <x>360</x>
         <y>330</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text" >
         <string>Counts</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Asymmetry</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Logorithm</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="pairTablePlotChoice" >
       <property name="geometry" >
        <rect>
         <x>360</x>
         <y>570</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text" >
         <string>Asymmetry</string>
        </property>
       </item>
      </widget>
      <widget class="QTableWidget" name="groupTable" >
       <property name="geometry" >
        <rect>
         <x>20</x>
         <y>100</y>
         <width>531</width>
         <height>221</height>
        </rect>
       </property>
       <property name="rowCount" >
        <number>10</number>
       </property>
       <property name="columnCount" >
        <number>3</number>
       </property>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <column>
        <property name="text" >
         <string>Group&#xd;
(Name)</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>Detector IDs</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>Ndet</string>
        </property>
       </column>
      </widget>
      <widget class="QLabel" name="label_7" >
       <property name="geometry" >
        <rect>
         <x>20</x>
         <y>70</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font" >
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text" >
        <string>Group Table</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8" >
       <property name="geometry" >
        <rect>
         <x>20</x>
         <y>370</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font" >
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text" >
        <string>Pair Table</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MantidQt::MantidWidgets::MWRunFiles</class>
   <extends>QWidget</extends>
   <header>MantidQtMantidWidgets/MWRunFiles.h</header>
  </customwidget>
  <customwidget>
   <class>MantidQt::MantidWidgets::InstrumentSelector</class>
   <extends>QComboBox</extends>
   <header>MantidQtMantidWidgets/InstrumentSelector.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
